driverkit/config/dev/flatcar_stable_4.19.106-flatcar.yaml

```diff
@@ -0,0 +1,5 @@
+kernelrelease: 4.19.106-flatcar
+target: flatcar-stable
```

Comment:
Do the targets in driverkit need to be different for each flavor of Flatcar? There is currently 4 channels: stable, beta, alpha, edge.
I see that for CentOS, there is a single CentOS target, and the implementation is able to try different URLs for different releases. Is it possible to do something like that with an unique target "flatcar"?

Reply:
Yeah, it makes sense. if you are already spotted a pattern that the guys put in place then let's follow for flatcar too.